articles/multifactor-authentication/api/index.md

```diff
@@ -1,69 +1,53 @@
 ---
 title: Multifactor Authentication API
-description: Index of Available MFA APIs
+description: Overview of available multifactor authentication APIs
 ---
 
-# Multifactor Authentication (MFA) API
+# Multifactor Authentication API
 
-The Multifactor Authentication (MFA) API is a set of APIs that allows you to:
-
-* Enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token)
-* Programmatically enroll and manage user authenticators
+The Multifactor Authentication (MFA) API endpoints allow you to enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token), as well programmatically enroll and manage user authenticators.
 
 ## Requirements
 
-To use the MFA APIs you will need **an enabled MFA rule that sets Guardian as the MFA provider**. Fore more info on this see [Guardian for Administrators](/multifactor-authentication/administrator).
+Before you can use the MFA APIs, you'll need to:
+
+* Enable the MFA grant type for your application. You can enable the MFA grant by going to [Applications > Your Application > Advanced Settings > Grant Types](${manage_url}/#/applications) and selecting MFA.
+* Create a rule that sets Guardian as the MFA provider. For more information, see [Guardian for Administrators](/multifactor-authentication/administrator).
 
 If you are using the MFA API in conjunction with a [Token endpoint](/api/authentication#get-token), you must meet the requirements of the corresponding grant.
 
-## Enforcing MFA when interacting with the Token Endpoint
+## Multifactor authentication with the Token endpoint
 
 We have expanded MFA support on the Token endpoints to cover the following use cases:
```

Comment:
I really don't understand how this fits in in this doc. Looks completely out of context to me. Is this relevant to the API or MFA in general?